### PR TITLE
Add Modding Guide page and other critical SEO content

### PR DESCRIPTION
**Pull request by TheCzar1994**
Adds comparable modding guide as mentioned here: https://discord.com/channels/452928402203344908/752003683780395069/1200902289171427369
Minor grammatical changes and internal links to getting started page
Adds the other "Getting Started" guides into the navbar and changes names in navbar to "Get Started with..."
Also adds Timothy’s site map and canonical URL

### CODE REVIEW

File: src/collections/posts/getting-started.md [hunk 1]
```diff
@@ -10,38 +10,48 @@ image: /static/uploads/posts/beat-saber-square.jpg
...
 
-As BeastSaber, we want help you find great maps and such is the task of our curators.
+At BeastSaber, we want to help you find great maps and such is the task of our curators.
 
 Maps on BeatSaver can marked as curated which means one of the curators considered them fun. You can [filter maps on BeatSaver](https://beatsaver.com/?curated=true) to only show curated maps.
 
-Additionally, you can checkout the [maps of the week](/maps-of-the-week/1) which are recent maps the curator team has considered particularly noteworthy.
+Additionally, you can check out the [Maps of the Week](/maps-of-the-week/1) - which will almost always have the following characteristics:
 
-On our frontpage we also feature playlists, which are collections of maps that are all great and reviewed by the curators.
```
Comment (TimothyGillespie) [inline src/collections/posts/getting-started.md:39]: I think this is duplicate content with About Curation? So one could consider linking that and hinting at more details there.

File: src/collections/posts/getting-started.md [hunk 2]
```diff
@@ -10,38 +10,48 @@ image: /static/uploads/posts/beat-saber-square.jpg
...
+Sometimes, the fun is in competing. If that's you then you will want the respective mods for [BeatLeader](https://beatleader.xyz/) and [ScoreSaber](https://scoresaber.com/). Both of these provide leaderboards for you to compare with friends or the world on custom songs. Using the mods you'll be able to submit your own scores (with the replays), see the leaderboard in-game and even collect performance points (PP for short) which are used to rank you in the world.
 
 # Communities
 
-If you're here to mingle then you'll find many communities that might just be for you. We will be hosting an list of communities soon - so stay tuned!
+If you're here to mingle then you'll find many communities that might just be for you. We will be hosting a list of communities soon - so stay tuned! In the meantime, you can check out our smaller list of Community sites and servers at the top under "Community Hub".
```
Comment (TimothyGillespie) [inline src/collections/posts/getting-started.md:57]: I don't see why "Community" in "list of Community" is capitalized.

File: src/collections/posts/installing-the-mod-guide-necessary-for-any-custom-songs.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    <img src="/uploads/mod-assistant-github.png">
+</p>
+
+This will download a small `ModAssistant.exe` file that you can place anywhere on your computer for future use and doesn’t require installing.
+
+2. When you first launch Mod Assistant, it will also try to automatically locate where you installed your game. If it can’t find it, you will have to manually locate the folder.
+
+    - Oculus User folder paths tend to be: `C:\Program Files\Oculus\Software\Software\hyperbolic-magnetism-beat-saber\`
+
+    - Steam User folder paths tend to be: `C:\Program Files (x86)\Steam\steamapps\common\Beat Saber\`
```
Comment (TimothyGillespie) [inline src/collections/posts/installing-the-mod-guide-necessary-for-any-custom-songs.md:38]: Should "User" be capitalized in both bullet points? I could see why, but then I find it a bit odd that Folder is not also capitalized as they kinda form one "unit".

File: src/collections/posts/installing-the-mod-guide-necessary-for-any-custom-songs.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+4. Once you agree, you’ll be able to click on the “Mods” button. All of the essential or “Core" mods will be pre-selected, but you can add as many other mods as you'd like in addition to the Core mods. When ready you can just click on the “Install or Update” button.
+
+    > ⚠️ IMPORTANT: You will need to have run Beat Saber at least once after installing the game before installing mods.
+
+<p align="center">
+    <img src="/uploads/mod-assistant-mods-install.png">
+</p>
+
+5. That’s it! You can now launch your game and enjoy all the wonderful mods that the community has built! You can always open ModAssistant again later to install more mods. We recommend against installing every mod at first as you’ll likely be overwhelmed by all the new options - and you may run into issues. Instead, take the time to learn about what each mod does. Click on the “Mod Info” button after selecting a mod to read more about it.
```
Comment (TimothyGillespie) [inline src/collections/posts/installing-the-mod-guide-necessary-for-any-custom-songs.md:54]: Use of a curly quotation mark. Not sure if that is intended so just pointing that out.

File: src/lib/Navbar.svelte [hunk 5]
```diff
@@ -33,14 +33,22 @@
...
           href: '/getting-started/custom-songs',
-        }
-      ]
+        },
+        {
+          name: 'Get Started with Custom Mods',
+          href: '/installing-the-mod-guide-necessary-for-any-custom-songs',
+        },
+        {
+          name: 'Get Started with Mapping',
```
Comment (TimothyGillespie) [inline src/lib/Navbar.svelte:48]: I did forget to raise this point in the last PR:
I think the sub menu names are too long. How about something like `Get Started` for the menu and then sub menus can be just `With Beat Saber` or even shorter `Beat Saber`. It think it'd be preceived as one unit due to the hierachy that is conciously interacted with prior + the intend for the subemenu names.

**Review by TimothyGillespie — APPROVED** [review]
I left some remarks. I'll leave it up to you if you want to address them or whether you have a differing opinion or I'm misunderstanding something. Thus, approval.

Thanks for adding the two pages and improving on my article btw! Very useful and much appreciated.

**Comment by TimothyGillespie** [comment]
LGTM

**Review by Pixelguymm — APPROVED** [review]

**Pull request merged**